chapters/220_persona-data-in-ctx-big-data.md

```diff
@@ -29,97 +28,95 @@ involved in such a knowledge discovery process, can be outlined as follows
 [@web_2013_big-data-collection-collides-with-privacy-concerns], whether this data is mandatory
-in that process or additionally obtained to for example measure and analyse customer behaviour. 
+in that process or additionally obtained to,for example, measure and analyze customer behaviour. 
 Common data points involved in big data analytics start with gender, age, residency or income, 
-goes on with time series events like changing current geo-location or web search history and goes 
-all the way up to health data and self-created content like posts, images or videos.
-Depending on the data point though, that data might not be that easy to collect. In general, most 
+goes on with time series events like changing current geo-location or web search history and continues 
+all the way up to health data and self-created content like posts, images, or videos.
+Depending on the data point though, those data might not be easy to collect. In general, most 
 businesses obtain data from within their own platforms. Some data might even be in the customer's 
-rang of control (e.g. customer or profile data), but most of the data is created during direct 
+range of control (e.g. customer or profile data), but most of the data is created during direct 
 (content creation, inputs) or indirect (transactions, meta information) interaction with the
-business. The sensitivity level of involved personal data is determined by how big is the benefit 
-for the customer in comparison to what is the vendor's demand from the customer's commitment (e.g. 
-required inputs, or usage requires access to location information) 
+business. The sensitivity level of involved personal data is determined by how big the benefit is
+for the customer in comparison to what the vendor's demand is from the customer's commitment (e.g. 
+required inputs, or usage requires access to location information).
 
 From a technical perspective, collecting indirectly created data is as simple as integrating 
 logging or debugging statements in the program logic. Since most vendors and organisations nowadays 
 have a business that is partially happening through the internet or is even completely based on it, 
-most scenarios of transactions utilizing server-client architectures.
-Furthermore, the *always-on* philosophy evolved to an imperative and implicit state of devices. 
+most scenarios of transactions utilize server-client architectures.
+Furthermore, the *always-on* philosophy has evolved to an imperative and implicit state of devices. 
 Standalone software, installed on a personal computer, calls the vendor's infrastructure that is 
-located in the cloud on a regularly basis, just to make sure that its user behaves properly, while
-en passant the vendor is provided with detailed user statistics. The web-browser already invokes a 
-common narrative that requests happening here and then - still preventable though But when it comes 
```

Comment:
let me rephrase it to "The web-browser  already __refects/practices__ a common narrative that requests happening here and then in the background, __although__ those requests are still preventable".